### ISSUE TITLE
PDF size incorrect

### ISSUE THREAD

**Issue by tresf** [qzind/tray#153]
Since https://github.com/qzind/tray/commit/a4171b07353ada2291fa74a0e8db7a3b1f383c2d, PDF printing has been slightly off, especially when using `scaleContent: false`

This is noticeable with an A4 PDF that is created to reach near the edges of the paper.  Prior to https://github.com/qzind/tray/commit/a4171b07353ada2291fa74a0e8db7a3b1f383c2d, the A4 scaled properly. Afterward it adds about 0.25 in margins around the entire document.

Reverting to version 2.0.1 temporarily resolves this issue.

**Comment by bberenz** [comment]
Since the Book object seems to print correctly, I reverted back to using it instead of the pdf merger, and fixed the problem with multiple pages not printing a different way.
Available under 054c9eb33fa6e414fd3f133e30057cb83ce50ed5

**Comment by tresf** [comment]
A4 test results still fail (identically) with https://github.com/qzind/tray/commit/054c9eb33fa6e414fd3f133e30057cb83ce50ed5 applied.

**Comment by tresf** [comment]
Quoting @bberenz:

> So stepping through the sources again with the old working code I noticed that both interfaces end up at the same point, but printable does two things pageable doesn't, either of which could be our issue.
Most likely is one of the overloaded setters uses a default page format.
> 
> 1. Change this line `job.setPrintable(bundle);`
> to this: `job.setPrintable(bundle, page);`
> Each page was being passed the format later, but as this is the base one, I can see it being the root of the problem.
> 2. Secondly, and only if the first solution doesn't work:
> Change the line `job.setPrintable(bundle);` into this block: 
   ```
   Book b = new Book();
   for(int i = 0; i < bundle.getNumberOfPages(); i++) {
       b.append(bundle, page);
   }
   job.setPageable(b);
   ```

> This assumes the problem is in the custom book object used inside the printerjob implementation, so we wrap our wrapped book in another book..

I'll test this out this week and report back.

**Comment by tresf** [comment]
Test results:

⛔️ Bullet 1 `job.setPrintable(bundle, page);` fails.
✅ Bullet 2 `job.setPageable(<book wrapper>);` passes.

**Comment by bberenz** [comment]
Working test case committed under e5d2af72f346020136f7877a646ac9f672c45d92

**Comment by tresf** [comment]
Closed via e5d2af7.  This patch will be available in 2.0.3-stable release.
